cli/cmd/brokers.go

```diff
@@ -19,16 +41,39 @@ var brokersCmd = &cobra.Command{
 	Long:  "Manage brokers associated with a service controller",
 }
 
-var createCmd = &cobra.Command{
+var createBrokersCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Add a service broker to service controller",
 	Long:  "Add a service broker to service controller",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return fmt.Errorf("Not implemented yet")
+		if len(name) == 0 {
+			return fmt.Errorf("need name of the broker")
+		}
+		if len(brokerURL) == 0 {
+			return fmt.Errorf("need brokerURL of the broker")
+		}
+		req := model.CreateServiceBrokerRequest{
+			Name:      name,
+			BrokerURL: brokerURL,
+		}
+		if len(user) > 0 {
+			req.AuthUsername = user
+		}
+		if len(password) > 0 {
+			req.AuthPassword = password
+		}
+		if len(spaceGUID) > 0 {
+			req.SpaceGUID = spaceGUID
+		}
+		body, err := json.Marshal(req)
+		if err != nil {
+			return err
+		}
+		return callService(BROKERS_URL, "POST", "create broker", ioutil.NopCloser(bytes.NewReader(body)))
 	},
 }
 
-var describeCmd = &cobra.Command{
+var describeBrokersCmd = &cobra.Command{
```

Comment:
so this one would be: `sc brokers describe NAME`.
Actually, this one is kind of odd. The help doesn't doesn't show a -n option, nor does it show NAME as an arg.  When I use -n I get an error, bad flag. When I use just NAME w/o -n then I don't get an error, just the help text.